invoker/function-maven-plugin/src/main/java/com/google/cloud/functions/plugin/DeployFunction.java

```diff
@@ -201,10 +201,28 @@
   @Parameter(alias = "deploy.envvarsfile", property = "function.deploy.envvarsfile")
   String envVarsFile;
 
+  /**
+   * List of key-value pairs to set as build environment variables. All existing environment variables
```

Comment:
Do you know if we set any build env vars today? I could see this overwrite behavior causing problems. An alternative is to append the user's env vars at the end of any existing env vars. If there's a collision, the last one appended should take precedence.

Reply:
Well, today, the maven plugin does nothing with this settings, so having this behavior of deleting previous values would be a clean no op.